docs/advanced-latedelivery.md

```diff
@@ -23,17 +23,17 @@ Then head to the `Template` tab in the `Contract Text` section which shows the n
 
 ![Advanced-Late-2](assets/advanced/late2.png)
 
-When the text in the `Template` tab is in sync with the text in the `Test Contract` tab, this results in a valid contract instance in the `Contract Data` tab. The contract data is represented using the JSON format, and contains the value of the variables declared in the contract template. For instance, the value for the `buyer` variable is `Betty Buyer`, highlighted in red:
+When the text in the `Template` tab is in sync with the text in the `Test Contract` tab, this results in a valid contract instance in the `Contract Data` tab. The contract data is represented using the JSON format and contains the value of the variables declared in the contract template. For instance, the value for the `buyer` variable is `Betty Buyer`, highlighted in red:
 
 ![Advanced-Late-3](assets/advanced/late3.png)
 
-Changes to the variables in the `Test Contract` are reflected in the Contract Data in real time, and vice versa. For instance, change `Betty Buyer` to a different name in the contract text to see the `partyId` change in the contract data.
+Changes to the variables in the `Test Contract` are reflected in the Contract Data in real-time, and vice versa. For instance, change `Betty Buyer` to a different name in the contract text to see the `partyId` change in the contract data.

```

Comment:
This can be reverted.

Reply:
Why is it `real-time` in one place and `realtime` in another in this PR?

Reply:
sorry, my mistake I fixed it now.